Exception error on Linux

I downloaded the latest release of durian and attempted to run it. I was instantly given an exception error, and the program then did nothing else.
I also tried version 0.1, which had the same result.

Reproduction and error:
```
dryvnt:durian/ $ java-jar durian-v0.1.1.jar
Exception in Application start method
Exception in thread "main" java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$152(LauncherImpl.java:182)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at net.thirdy.durian.util.config.ItemWatchConfig$ItemWatch.fromConfig(ItemWatchConfig.java:92)
	at net.thirdy.durian.util.config.ItemWatchConfig.lambda$3(ItemWatchConfig.java:37)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
	at net.thirdy.durian.util.config.ItemWatchConfig.loadItemWatchList(ItemWatchConfig.java:38)
	at net.thirdy.durian.ui.DurianApplication.start(DurianApplication.java:102)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$159(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$172(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$170(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$171(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$null$48(GtkApplication.java:139)
	... 1 more
```

--- a/durian/src/main/java/net/thirdy/durian/util/config/ItemNamesConfig.java
+++ b/durian/src/main/java/net/thirdy/durian/util/config/ItemNamesConfig.java
@@ -18,6 +18,8 @@ public class ItemNamesConfig implements Config {
 	public String fileName() {
 		return "itemnames.txt";
 	}
+	
+	private static final String regexString = "^;.+|.+";
 
 	public List<ItemChoice> loadNamesList() {
 		try {
@@ -30,7 +32,7 @@ public class ItemNamesConfig implements Config {
 				String defaultRawConfig = loadDefault();
 				list = Arrays.asList(defaultRawConfig.split(System.lineSeparator()))
 						.stream()
-						.filter(s -> !s.isEmpty() && !s.startsWith(";"))
+						.filter(s -> s.matches(regexString))
 						.map(s -> ItemChoice.fromConfig(s))
 						.collect(Collectors.toList());
 				

--- a/durian/src/main/java/net/thirdy/durian/util/config/ItemWatchConfig.java
+++ b/durian/src/main/java/net/thirdy/durian/util/config/ItemWatchConfig.java
@@ -21,6 +21,8 @@ public class ItemWatchConfig implements Config {
 	public String fileName() {
 		return "itemwatch.txt";
 	}
+	
+	private static final String regexString = "^;.+|.+|.+|.+";
 
 	public List<ItemWatch> loadItemWatchList() {
 		try {
@@ -33,7 +35,7 @@ public class ItemWatchConfig implements Config {
 				String defaultRawConfig = FileUtil.fromClasspath(ItemWatchConfig.class, "default-itemwatch.txt");
 				list = Arrays.asList(defaultRawConfig.split(System.lineSeparator()))
 						.stream()
-						.filter(s -> !s.isEmpty() && !s.startsWith(";"))
+						.filter(s -> s.matches(regexString))
 						.map(e -> ItemWatch.fromConfig(e))
 						.collect(Collectors.toList());
 				

